src/rtc.c

```diff
@@ -426,6 +424,41 @@ void FormatDecimalTimeWithoutSeconds(u8 *txtPtr, s8 hour, s8 minute, bool32 is24
     *txtPtr = EOS;
 }
 
+u16 GetFullYear(void)
+{
+    struct DateTime dateTime;
+    RtcCalcLocalTime();
+    ConvertTimeToDateTime(&dateTime, &gLocalTime);
+
+    return dateTime.year;
+}
+
+enum Month GetMonth(void)
+{
+    struct DateTime dateTime;
+    RtcCalcLocalTime();
+    ConvertTimeToDateTime(&dateTime, &gLocalTime);
+
+    return dateTime.month;
+}
+
+u8 GetDay(void)
+{
+    struct DateTime dateTime;
+    RtcCalcLocalTime();
+    ConvertTimeToDateTime(&dateTime, &gLocalTime);
+
+    return dateTime.day;
+}
+
+enum Weekday GetDayOfWeek(void)
+{
+    struct DateTime dateTime;
+    RtcCalcLocalTime();
+    ConvertTimeToDateTime(&dateTime, &gLocalTime);
+
+    return dateTime.dayOfWeek;
+  
 enum TimeOfDay TryIncrementTimeOfDay(enum TimeOfDay timeOfDay)
```

Comment:
```suggestion
enum TimeOfDay UNUSED TryIncrementTimeOfDay(enum TimeOfDay timeOfDay)
```

Reply:
idk why the PR won't merge without this now... can you add `UNUSED` to `TryDecrementTimeOfDay` too and then I'll merge?

Reply:
btw `TryDecrementTimeOfDay` and `TryIncrementTimeOfDay` are used for the Pokedex time of day functionality, setting them to unused may cause problems

Reply:
I think i figured it out and I think i'm just dumb